docs/source/getting_started/content/con_understanding-codeready-containers-configuration.adoc

```diff
@@ -0,0 +1,10 @@
+[id="understanding-codeready-containers-configuration_{context}"]
+= Understanding {prod} configuration
+
+Both the [command]`{bin}` binary and the {prod} virtual machine can be configured using the [command]`{bin} config` command.
+The [command]`{bin} config` command requires a subcommand to act on the configuration.
+The available subcommands are `get`, `set,` `unset`, and `view`.
+The `get`, `set`, and `unset` subcommands operate using named configurable properties.
+
+Preflight checks for the [command]`{bin} start` and [command]`{bin} setup` commands can be configured to be skipped or to report a warning rather than an error.
+The configurable properties for preflight checks begin with `skip-check` or `warn-check`, respectively.
```

Comment:
I don't understand the technical concept here, so discard this command if it's out of place...

I'm commenting on "begin with". If you have a bunch of configurable properties, there is no real beginning or end. A list of properties _does_ have a beginning and end, or if you have to define them in a certain order.

Reply:
"Begin with" is referring to the names of configurable properties. A relatively large number of them exist for the pre-start (or "preflight") checks that are run on `crc setup` and `crc start`. All of the names of properties related to these checks start with `skip-check` or `warn-check`.

For example, here are some of the properties:
 * skip-check-bundle-cached
 * warn-check-bundle-cached
 * skip-check-crc-dnsmasq-file
 * warn-check-crc-dnsmasq-file
 * skip-check-crc-network
 * warn-check-crc-network
 * skip-check-crc-network-active
 * warn-check-crc-network-active

In this case, we can probably clarify that simply by stating that "The names of configurable properties for pre-start checks begin with `skip-check` or `warn-check`."

The "respectively" regards whether the check is skipped entirely or reports a warning rather than an error.

Reply:
Ah...so I get it.

So how about this:

> You can configure the [command]`{bin} start` and [command]`{bin} setup` commands such that the preflight check skips or reports a warning rather than an error.

> To configure the commands, add one of the following prefixes to each property:

> `skip-check`::Skips the preflight check. For example: `skip-check-bundle-cached`
> `warn-check`::Reports a warning rather than an error. For example: `warn-check-bundle-cached`

> For example:
F